tests/entrypoints/test_openai_server.py

```diff
@@ -1342,6 +1340,7 @@ async def test_batch_embedding(embedding_server, client: openai.AsyncOpenAI,
     assert embeddings.usage.total_tokens == 17
 
 
+@pytest.mark.asyncio
```

Comment:
Need to update this test to actually check Chat Completions API (The code is currently only checking Completions API)

Reply:
Would you prefer these tests to be checked within the same function, or is it better to keep them separate? Currently, I have split them into two functions.
@DarkLight1337

Reply:
Keep them in separate functions. I'm planning to split up this file in a later PR.